test/image/mocks/gl3d_mesh3d_surface3d_scatter3d_line3d_error3d_log_reversed_ranges.json

```diff
@@ -95,7 +95,7 @@
   ],
   "layout": {
     "height": 758,
-    "width": 1310,
+    "width": 1000,
```

Comment:
Why did you have to change these `layout.width` values?

Reply:
Actually we don't need to trim the white spaces anymore. On #3571 where I used to increase the `pixelRatio` when the `antialias is not set by hardware, there was some strange behaviour on the CI test that stretched one axis to possibly stay within the maximum width/height size limits. Here is an example showing that effect: https://github.com/plotly/plotly.js/blob/7897dc272466ef12676b0ac55cf2bac99031037d/test/image/baselines/gl3d_surface-circular-colorscale.png

Reply:
Thanks for the info!

Can we then bring back those `layout.width` to their previous values?